tests/test_doc_auto_generation.py

```diff
@@ -355,36 +355,5 @@ def test_doc_lists(docs_descriptor):
     assert markdown(docstring) == markdown(docs_descriptor['result'])
 
 
-dummy_docstring = """Multiplies 2 tensors (and/or variables) and returns a *tensor*.
```

Comment:
Why is this being removed?

Reply:
This was there to check for a very specific bug, not just code inside a docstring. I don't think we should remove it. See #11508. The documentation generation is bug-prone, we should have as many tests as possible, they are very cheap to run.

Reply:
Alright, I removed these test deletion